src/MatchingResult.php

```diff
@@ -86,6 +72,9 @@ public function methods(): array
         return $this->methods;
     }
 
+    /**
+     * @psalm-assert-if-true true $this->isSuccess()
```

Comment:
```suggestion
     * @psalm-assert-if-true true $this->isSuccess()
```

Why it? `psalm-assert-if-true` use for methods that boolean returns.